src/EventManagement.Services/PowerOffice/PowerOfficeService.cs

```diff
@@ -6,14 +8,16 @@
 using GoApi.Invoices;
 using GoApi.Party;
 using losol.EventManagement.Domain;
+using losol.EventManagement.Infrastructure;
 using Microsoft.Extensions.Options;
 
 namespace losol.EventManagement.Services.PowerOffice {
     public class PowerOfficeService : IInvoicingService {
 
         private readonly Go api;
+        private readonly ApplicationDbContext _db;
 
-        public PowerOfficeService (IOptions<PowerOfficeOptions> options) {
+        public PowerOfficeService (IOptions<PowerOfficeOptions> options, ApplicationDbContext db) {
```

Comment:
@losviko `ApplicationDbContext` shouldn't be accessed from this class. This class shouldn't have any business logic, and should only perform PowerOffice API related actions. Mixing the two will result in duplicate code when introducing alternate `IInvoicingService` implementations.